AnimationProfiles/TPP/tpp_AP.pddl

```diff
@@ -0,0 +1,145 @@
+(define (animation TPP-Propositional)
+
+; Defines the Animation profile for the TPP propositional problem
+; 18/08/2023
+; By Shu Tong Luo
+;
+; Note: Given the nature of the problem domain, multiple depots / markets 
+; are visualised as overlapping as there is no initial state predicate.
+ 
+ 
+; Specifies the goods is loaded onto a truck at a given level
+   (:predicate loaded
+                  :parameters (?g ?t ?l)
+                  :effect(
+                  (equal (?t y) (add (?l y) 50))
+                  (equal (?g y) (add (?t y) 20))
+                  (assign (?g x) (function align_middle (objects ?g ?t)))
+                  )
+   )
```

Comment:
The current proposed visualization can mislead the meaning of the loaded predicate. As it is now, it seems that the track travels to another location, whereas this indicates that a certain good is loaded, and the level ?l stands for how many of these goods are loaded in the truck. 

I'm keen to see if you can come up with an alternative visualisation. It is not trivial.